doc/attr.xml

```diff
@@ -1325,21 +1510,31 @@ fail]]></Example>
   <Attr Name="DigraphReverseAttr" Arg="digraph"/>
   <Returns>A digraph.</Returns>
   <Description>
-    If <A>digraph</A> is a digraph, then this operation returns a digraph
+    If <A>digraph</A> is an immutable digraph, then this operation returns a new digraph
```

Comment:
Could you write "new immutable digraph" rather than just "new digraph" please.

Also, I would prefer the logic of this documentation to be changed somewhat. It currently says basically:

> "if X is immutable, then this is what the function does. if X is mutable, then this function changes the digraph in place".

Technically it's not actually saying what the function _does_ for a mutable digraph, if you know what I mean.

Perhaps something along the lines of "The reverse of a digraph is BLAH. This operation returns the reverse of the digraph ARG. If ARG is immutable then, ..., otherwise if ARG is mutable, ...".